packages/twenty-server/src/engine/integrations/file-storage/utils/read-file-content.ts

```diff
@@ -0,0 +1,11 @@
+import { Readable } from 'stream';
+
+export const readFileContent = async (stream: Readable): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const chunks: Buffer[] = [];
+
+    stream.on('data', (chunk) => chunks.push(Buffer.from(chunk)));
+    stream.on('end', () => resolve(Buffer.concat(chunks).toString('utf8')));
+    stream.on('error', (err) => reject(err));
+  });
+};
```

Comment:
Another way to write this code is by using a [`for await ... of` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of), which should automatically forward errors.

```suggestion
export const readFileContent = async (stream: Readable): Promise<string> => {
  const chunks: Buffer[] = [];
  
  for await (const chunk of stream) {
    chunks.push(Buffer.from(chunk))
  }
  
  return Buffer.concat(chunks).toString('utf8')
};
```